mappings/net/minecraft/util/registry/DynamicRegistryManager.mapping

```diff
@@ -0,0 +1,48 @@
+CLASS net/minecraft/class_5455 net/minecraft/util/registry/DynamicRegistryManager
+	FIELD field_25918 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_25919 INFOS Ljava/util/Map;
+	METHOD method_30518 getDimensionType ()Lnet/minecraft/class_2378;
```

Comment:
This one is a little bit confusing, it sounds like you're getting a dimension type rather than a registry of them.

Reply:
Hmm? I thought this is okay as we have like `getBlocks` for block tags in like the tag holder. `registryManager.getDimensionType` should be clear?

Reply:
Following that pattern, it would be `getDimensionTypes` (plural)

Reply:
Plural sounds good to me if it's a registry that's being returned here.